### PR TITLE
[kr] 'mutate'의 번역을 기존 '반영'에서 '변경'으로 수정했습니다.

### PR DESCRIPTION
**Pull request by hank-smatoos**

### CODE REVIEW

File: docs/kr/intro.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ new Vue({
 그러나 **공통의 상태를 공유하는 여러 컴포넌트** 가 있는 경우 단순함이 빠르게 저하됩니다.
 
 - 여러 뷰는 같은 상태에 의존합니다.
-- 서로 다른 뷰의 작업은 동일한 상태를 반영해야 할 수 있습니다.
+- 서로 다른 뷰의 작업은 동일한 상태를 변경해야 할 수 있습니다.
```
Comment (ChangJoo-Park) [inline docs/kr/intro.md]: "반영" and "변경" is similar. but in this sentence, mutation is meaning "active action".

So it need to change to "변경" for understand

**Comment by ChangJoo-Park** [comment]
@kazupon Thank you for ping me 👍

**Comment by kiaking** [comment]
@hank-smatoos @ChangJoo-Park Hi! Pretty old PR but do you know what should we do with this? I don't speak Korean so I have no idea what it's fixing... :p

**Comment by kiaking** [comment]
Seems like this PR is pretty much abandoned. I'll be closing this one, though please feel free to recreate the PR against the latest docs! 🤝

**Pull request closed without merging**